bootstrap.py

```diff
@@ -209,7 +208,8 @@ def register_systems(org_name, activationkey, release):
     options.smargs += " --serverurl=https://%s:%s/rhsm --baseurl=https://%s/pulp/repos" % (options.foreman_fqdn, API_PORT, options.foreman_fqdn)
     if options.force:
         options.smargs += " --force"
-    exec_failexit("/usr/sbin/subscription-manager register --org '%s' --name '%s' --activationkey '%s' %s" % (org_label, FQDN, activationkey, options.smargs))
+    # RHEL 5 sometimes register successfully with "Error updating system data on the server" error
```

Comment:
can we make that rhel5 only? (or fix sub-man in rhel5? :))
I think allowing sub-man to fail randomly and ignoring it is a bad idea for a tool that is supposed to subscribe the machine